src/server/game/Entities/Player/Player.cpp

```diff
@@ -18672,31 +18672,31 @@ bool Player::Satisfy(AccessRequirement const* ar, uint32 target_map, bool report
 
 bool Player::CheckInstanceLoginValid(Map* map)
 {
-    if (!map->IsDungeon() || IsGameMaster())
+    if (!map->IsDungeon() || IsInstanceLoginGameMasterException())
```

Comment:
That will duplicate the group checks, the idea behind IsInstanceLoginGameMasterException() is to replace any previous "return false;" in Player::CheckInstanceLoginValid() .
I'd like to merge this PR without the changes in Player::CheckInstanceLoginValid(), the rest of the PR is ok